src/cli.ts

```diff
@@ -59,20 +62,18 @@ async function runLinuxInstall(
   return destinationFolder
 }
 
-async function MacOSDeps(file: string): Promise<void> {
-  fs.readFile(file, 'utf-8', function (err, data) {
-    if (err) throw err
-
-    const newValue = data.replace(
-      /install -D/gim,
-      '$(brew --prefix coreutils)/bin/ginstall -D'
-    )
+async function MacOSDeps(file: string, githubToken?: string): Promise<void> {
+  const linuxScript = String(fs.readFileSync(file, 'utf-8'))
+  const macosScript = linuxScript.replace(
+    /install -D/gim,
+    '$(brew --prefix coreutils)/bin/ginstall -D'
+  )
+  fs.writeFileSync(file, macosScript, 'utf-8')
 
-    fs.writeFile(file, newValue, 'utf-8', function (e) {
-      if (e) throw e
-    })
-  })
-  await exec.exec(`brew install coreutils`)
+  const env = githubToken
+    ? {env: {HOMEBREW_GITHUB_API_TOKEN: githubToken}}
+    : undefined
+  await exec.exec('brew', ['install', 'coreutils'], env)
```

Comment:
I wonder if this function can be simplified to `brew install clojure/tools/clojure` execution